packages/angular_devkit/build_angular/src/builders/browser/specs/svg_spec.ts

```diff
@@ -57,7 +57,29 @@ describe('Browser Builder allow svg', () => {
         host.scopedSync().read(join(outputPath, 'main.js')),
       );
 
-      expect(content).toContain('ɵɵnamespaceSVG');
+      // Extract the AppComponent template function.
+      // Example function:
+      // function AppComponent_Template(rf, ctx) {
+      //   if (rf & 1) {
+      //     _angular_core__WEBPACK_IMPORTED_MODULE_1__.ay();
+      //     _angular_core__WEBPACK_IMPORTED_MODULE_0__.dZ(0, "svg", 0)(1, "text", 1);
+      //     _angular_core__WEBPACK_IMPORTED_MODULE_0__.fH(2, "Hello World");
+      //     _angular_core__WEBPACK_IMPORTED_MODULE_0__.dY()();
+      //   }
+      // }
+
+      const templateFnMatch = content.match(
+        /function AppComponent_Template\(rf, ctx\)\s*\{[\s\n]*if \(rf & 1\) \{\n([\s\S]+?)\n[ ]*\}/,
+      );
+      expect(templateFnMatch).toBeDefined();
+      const [, templateCreate] = templateFnMatch!;
+
+      const createLines = templateCreate.split('\n');
+      // Zero argument function first - that'd be ɵɵnamespaceSVG.
+      expect(createLines[0]).toMatch(/\b[^(]+\(\);$/);
+      // Create element function second.
+      expect(createLines[1]).toMatch(/\b[^(]+\(0, "svg", 0\)/);
```

Comment:
Consider: Do we care this much about the generated output? Could we get away with something simpler like `expect(content).toContain('"svg"');`?

Reply:
We can probably just check for `expect(content).toContain('Hello world');` since what we are testing is that the contents of the template has been inlined.

Reply:
That does sound much simpler! 😄 I overindexed on the previous check. Thanks for that context, will go for the "Hello World" check.